src/enigma2/Timers.cpp

```diff
@@ -1192,7 +1206,14 @@ bool Timers::TimerUpdatesRegular()
 
   Logger::Log(LEVEL_DEBUG, "%s No of timers: removed [%d], untouched [%d], updated '%d', new '%d'", __FUNCTION__, iRemoved, iUnchanged, iUpdated, iNew);
 
-  std::vector<EpgEntry> timerBaseEntries(m_timers.begin(), m_timers.end());
+  std::vector<EpgEntry> timerBaseEntries;
+  for (auto& timer : m_timers)
+  {
+    EpgEntry entry = static_cast<EpgEntry&>(timer);
```

Comment:
https://github.com/kodi-pvr/pvr.vuplus/blob/Matrix/src/enigma2/data/Timer.h#L45

If a `Timer` is an `EpgEntry`, for what do you need the cast?